tmf/org.eclipse.tracecompass.tmf.ui/src/org/eclipse/tracecompass/tmf/ui/views/histogram/HistogramView.java

```diff
@@ -610,6 +610,11 @@ private void setNewRange(long startTime, long duration) {
             realStart = fTraceStartTime;
         }
 
+        if (duration == 1) {
```

Comment:
let's do <= 1? that way we can catch 0 too?

Reply:
Good idea 👍